src/main/java/axoloti/utils/Preferences.java

```diff
@@ -200,6 +208,37 @@ public String getCurrentFileDirectory() {
         return CurrentFileDirectory;
     }
 
+    public IPlatform getPlatform(Patch p) {
+        IPlatform target = null;
+        if (Platform != null && !Platform.isEmpty()) {
+            try {
+                Logger.getLogger(Preferences.class
+                        .getName()).log(Level.INFO, "Attempting to load target {0}", Platform);
+                ClassLoader cl = Axoloti.class.getClassLoader();
+                Class c = cl.loadClass(Platform);
+                Object o = c.newInstance();
```

Comment:
No, dynamic class loading is useful as it a new platform can be included in a separate JAR without any change of code in the application. 
If I want to preset preferences, then its possible to implement a registration system, such that 'known' platforms are register, and so available for the user.
until there are more official platforms, I don't want this in the preferences panel, hence why Ive not done this at this time.